resources/web/wwi/WebotsAnimation.js

```diff
@@ -24,10 +24,13 @@ export default class WebotsAnimation extends HTMLElement {
         return prefix + path;
       }`;
     document.head.appendChild(script);
-    let name = document.getElementsByTagName('webots-animation')[0].title;
-    if (!name)
-      name = location.pathname.substring(location.pathname.lastIndexOf('/') + 1).replace('.html', '');
-    this.setNames(name);
+    let x3d = this.getAttribute('x3d');
+    let json = this.getAttribute('json');
```

Comment:
Can't we use consts here?
```suggestion
    const x3d = this.getAttribute('x3d');
    const json = this.getAttribute('json');
```